packages/libs/react-ui/src/components/Select/Select.css.ts

```diff
@@ -1,23 +1,56 @@
+import { vars } from '../../styles';
+
 import { sprinkles } from '@theme/sprinkles.css';
-import { style } from '@vanilla-extract/css';
+import { style, styleVariants } from '@vanilla-extract/css';
 
 export const containerClass = style([
   sprinkles({
-    backgroundColor: '$neutral1',
-    color: '$neutral5',
-    padding: '$2',
     borderRadius: '$sm',
+    paddingRight: '$4',
+    backgroundColor: {
+      lightMode: '$white',
+      darkMode: '$gray100',
+    },
+    color: {
+      lightMode: '$gray60',
+      darkMode: '$gray40',
+    },
+    borderColor: {
+      lightMode: '$white',
+      darkMode: '$gray60',
+    },
   }),
   {
     position: 'relative',
-    border: 'solid 1px',
   },
 ]);
+
 export const containerClassDisabled = style([
-  sprinkles({ pointerEvents: 'none' }),
-  { opacity: 0.5 },
+  sprinkles({
+    backgroundColor: {
+      lightMode: '$gray30',
+      // darkMode: '$gray100',
+    },
+    color: {
+      lightMode: '$gray100',
+      // darkMode: '$gray40',
+    },
+  }),
 ]);
 
+export const selectVariants = styleVariants({
+  default: [
+    {
+      borderBottom: `1px solid ${vars.colors.$gray30}`,
+    },
+  ],
+  solid: [
+    {
+      border: `1px solid ${vars.colors.$gray30}`,
+    },
+  ],
```

Comment:
I guess it's between use case vs what to expect visually. I think it's the default one that would be used in a form though, right? The one I named solid shows up in the design as part of the header, so default would become form and the solid one would become default? But that doesn't make too much sense either. I initially used default and form but found it confusing. What do you think? Perhaps I'm wrong thinking that the current default one would be the form one..

Reply:
hmmm I see where you're coming from. So we can set the default variant prop to anything regardless of what we name it. I just think the form variant should be named form for clarity and then we could name the other one something descriptive as well. I do think you're right that it will most likely be used in a form so that can be the default variant in the end

Reply:
So now the default = form and we have flat as alternative. I'll resolve this conversation but we'll get approval from all parties involved before merging :)